src/olympia/devhub/tasks.py

```diff
@@ -143,17 +145,20 @@ def wrapper(task, id_, hash_, *args, **kw):
 
 
 @validation_task
-def validate_file_path(path, hash_, listed, **kw):
+def validate_file_path(path, hash_, listed=True, is_webextension=False, **kw):
```

Comment:
All calls to `validate_file_path` are always explicitly passing `listed` and it's analogous to `run_validator` where we default to `True` too.